lib/inspec/resources/oracledb_session.rb

```diff
@@ -39,9 +39,7 @@ def initialize(opts = {})
 
       @su_user = opts[:as_os_user]
       @db_role = opts[:as_db_role]
-
-      # we prefer sqlci although it is way slower than sqlplus, but it understands csv properly
-      @sqlcl_bin = "sql" unless opts.key?(:sqlplus_bin) # don't use it if user specified sqlplus_bin option
+      @sqlcl_bin = opts[:sqlcl_bin] || nil
```

Comment:
Is `opts[:sqlcl_bin]` going to be truthy, or is it going to be `"sql"`? It might be better as:

```ruby
@sqlcl_bin = "sql" if opts[:sqlcl_bin]
```